OpenRA.Mods.Common/Traits/LinkOnRepairCursor.cs

```diff
@@ -0,0 +1,49 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Send actor to link to repair cursor.")]
```

Comment:
Ok I think I understand what this is for now, this is not for providing the repair cursor when sending a unit to an actor to repair, but when hovering over a repairable unit, right? This makes more sense now, but why isn't it simply part of `RepairableLinkClient` without which it wouldn't properly work anyway?

Reply:
https://github.com/OpenRA/OpenRA/pull/20665#discussion_r1450788616